src/web/routes/application/phone-number/sanitize.js

```diff
@@ -0,0 +1,19 @@
+const { parsePhoneNumberFromString } = require('libphonenumber-js')
+
+// validation has already happened so no need for empty/null checks
+function getPhoneNumber (phoneNumber) {
+  const parsedNumber = parsePhoneNumberFromString(phoneNumber, 'GB')
+  return parsedNumber.number
+}
+
+const sanitize = (req, res, next) => {
+  // Do not mutate the phone number as users will want to see it in the format they entered it in on the check page
```

Comment:
extra word `it in on the check page` -> `it on the check page`

Reply:
Fixed.